### ISSUE TITLE
add missing growth patterns

### ISSUE THREAD

**Issue by k-doering-NOAA** [nmfs-stock-synthesis/test-models#25]
2, 3, 4, 5 are not represented.

**Comment by k-doering-NOAA** [comment]
Yes, Exactly! I was looking through the current test set and seeing what was not represented.

Another option besides making the version of simple is seeing if we can change an existing model that is already being run and runs quickly. That way, we make the model set more orthogonal, rather than having simple with all the same features but just 1 difference in MG options.

**Comment by Rick-Methot-NOAA** [comment]
With the approach I suggested, it would be more like a unit test of growth (or of all biology) that is run on request if the code change is anything in ss_biofxn.tpl.  A similar type of test could run all selectivity options if there is any change in ss_selex.tpl.

**Comment by k-doering-NOAA** [comment]
Hmm, that does make some sense. We could of course do both approaches, as I'm not sure which one will pan out to be easier. The unit test one sounds like it would take more time to me, but maybe I am overthinking it.

**Comment by Rick-Methot-NOAA** [comment]
Any "simple" run is rather quick.  So, I was thinking that the unit test runs I sketched would be an additional gha that is only invoked if certain files are changed.

**Comment by k-doering-NOAA** [comment]
Thanks, Rick, I think we are on the same page!

By "take more time", I meant time to develop - I agree the simple runs will run quick once they are set up!
